### ISSUE TITLE
Question about contribution policy

### ISSUE THREAD

**Issue by eebmagic** [2020PB/police-brutality#408]
Should reports only be made regarding police interactions with protestors? For example I found an incident that doesn't have a report listed, but it is not related to protests. 
The CONTRIBUTING.md only says "examples of excessive force being used by law enforcement officers during the 2020 protests sparked by the death of George Floyd". And this incident would be during these protests, but not at a protest or directly related to a protest except for the excessive police force.

Link to incident i'm referring to:
[https://www.washingtonpost.com/local/public-safety/fairfax-county-police-officer-charged-after-using-stun-gun-on-black-man-without-provocation-police-say/2020/06/07/4e7b4a90-a858-11ea-b619-3f9133bbb482_story.html?utm_source=reddit.com](https://www.washingtonpost.com/local/public-safety/fairfax-county-police-officer-charged-after-using-stun-gun-on-black-man-without-provocation-police-say/2020/06/07/4e7b4a90-a858-11ea-b619-3f9133bbb482_story.html?utm_source=reddit.com)

**Comment by haydenbspence** [comment]
I would also be interested to know if this would apply.

**Comment by emarcey** [comment]
@eebmagic 

We are trying to focus on LEO brutality during the George Floyd and related protests. Regarding the Fairfax incident and some similar others, we have legacied in with the "non-protest" tag, as we hope to expand beyond just protests in the future.

But other than that handful of extant issues, all incidents should be related to US protests. The [incident criteria](https://github.com/2020PB/police-brutality/blob/master/docs/incident-criteria.md) have been updated recently to better reflect the scope of the project.

Sorry for the delay. Just a few of us and a lot of brutality happening 😅
